client/app/components/form/UserOperatorMultiStepForm.tsx

```diff
@@ -95,9 +98,12 @@ export default function UserOperatorMultiStepForm({
         )}
         <MultiStepFormBase
           baseUrl={`/dashboard/operators/user-operator/${userOperatorId}`}
-          cancelUrl="/dashboard/operators"
+          // TODO: change cancelUrl for industry user depending on if they are creating or editing an operator
+          cancelUrl={`/dashboard/${
+            isCasInternal ? "operators" : "select-operator"
```

Comment:
This fix is unrelated to this PR though I noticed the `Cancel` button was broken for industrial users. Once #441 is in  we will likely need to make the link go elsewhere (maybe just the dashboard?) depending on form status.